speechbrain/nnet/RNN.py

```diff
@@ -1243,31 +1246,44 @@ def forward(self, x, hx: Optional[Tensor] = None):
 
         return h
 
-    def _ligru_cell(self, w, ht):
+    def _ligru_cell_cpu(self, w, ht, drop_mask):
```

Comment:
Why do we have a cpu and a non-cpu function? It's more of a normal forward or a cupy forward, no ? Naming might be misleading here